docs/cluster-agent/GETTING_STARTED.md

```diff
@@ -229,23 +229,36 @@ root@datadog-cluster-agent-8568545574-x9tc9:/# datadog-cluster-agent status
 
 #### On the Node Agent side
 
+You can check the status of the Datadog Cluster Agent while running the status command of the agent:
+`agent status`
+
+If the Datadog Cluster Agent is enabled, you should see:
+
+```
+[...]
+ =====================
+ Datadog Cluster Agent
+ =====================
+   Datadog Cluster Agent available at: 
+```
+
 Make sure the Cluster Agent service was created before the agents' pods, so that the DNS is available in the environment variables:
 
 ```
-root@datadog-agent-9d5bl:/# env | grep DCA_ | sort
-DCA_SERVICE_PORT=5005
-DCA_SERVICE_HOST=10.100.202.234
-DCA_PORT_5005_TCP_PORT=5005
-DCA_PORT=tcp://10.100.202.234:5005
-DCA_PORT_5005_TCP=tcp://10.100.202.234:5005
-DCA_PORT_5005_TCP_PROTO=tcp
-DCA_PORT_5005_TCP_ADDR=10.100.202.234
+root@datadog-agent-9d5bl:/# env | grep DATADOG_CLUSTER_AGENT | sort
+DATADOG_CLUSTER_AGENT_SERVICE_PORT=5005
```

Comment:
@CharlyF you'll need to update `datadog-cluster-agent_service.yaml` the service is still named `dca` there